plugins/utils/nvim-colorizer.nix

```diff
@@ -0,0 +1,166 @@
+{ pkgs, config, lib, ... }:
+with lib;
+let
+
+  cfg = config.plugins.nvim-colorizer;
+  helpers = import ../helpers.nix { inherit lib; };
+
+  colorizer-options = {
+    RGB = mkOption {
+      description = "#RGB hex codes";
+      type = types.nullOr types.bool;
+      default = null;
+    };
+    RRGGBB = mkOption {
+      description = "#RRGGBB hex codes";
+      type = types.nullOr types.bool;
+      default = null;
+    };
+    names = mkOption {
+      description = "\"Name\" codes like Blue or blue";
+      type = types.nullOr types.bool;
+      default = null;
+    };
+    RRGGBBAA = mkOption {
+      description = "#RRGGBBAA hex codes";
+      type = types.nullOr types.bool;
+      default = null;
+    };
+    AARRGGBB = mkOption {
+      description = "0xAARRGGBB hex codes";
+      type = types.nullOr types.bool;
+      default = null;
+    };
+    rgb_fn = mkOption {
+      description = "CSS rgb() and rgba() functions";
+      type = types.nullOr types.bool;
+      default = null;
+    };
+    hsl_fn = mkOption {
+      description = "CSS hsl() and hsla() functions";
+      type = types.nullOr types.bool;
+      default = null;
+    };
+    css = mkOption {
+      description = "Enable all CSS features: rgb_fn, hsl_fn, names, RGB, RRGGBB";
+      type = types.nullOr types.bool;
+      default = null;
+    };
+    css_fn = mkOption {
+      description = "Enable all CSS *functions*: rgb_fn, hsl_fn";
+      type = types.nullOr types.bool;
+      default = null;
+    };
+    mode = mkOption {
+      description = "Set the display mode";
+      type = types.nullOr (types.enum [ "foreground" "background" "virtualtext" ]);
+      default = null;
+    };
+    tailwind = mkOption {
+      description = "Enable tailwind colors";
+      type = types.nullOr (
+        types.oneOf [
+          types.bool
+          (types.enum [ "normal" "lsp" "both" ])
+        ]
+      );
+      default = null;
+    };
+    sass = mkOption {
+      description = "Enable sass colors";
+      type = types.nullOr (
+        types.submodule {
+          options = {
+            enabled = mkOption {
+              type = types.nullOr types.bool;
+              default = null;
+            };
+            parsers = mkOption {
+              type = types.nullOr types.attrs;
+              default = null;
+            };
+          };
+        }
+      );
+      default = null;
+    };
+    virtualtext = mkOption {
```

Comment:
mind adding a description here (and for the other ones in the sass colors)? Now with the work on the new doc site I wanted to make sure all of the options are properly documented

Reply:
Done :)